### ISSUE TITLE
Country, user field

### ISSUE THREAD

**Issue by darkarchon** [modxcms/revolution#12475]
Weird thing in MODx is a user field - country.
It has stored in database by country name, on language that you add/edit field. And if you change backend language the country name doesn't change.
Preferred method is store code - alpha-2, alpha-3 or better is numeric(all from ISO-3366-1)
This field, if I'm right is for international stuff, but for now is not useful at all.

P.S. The user fields must be more flexible.

**Comment by pixelchutes** [comment]
Duplicate of #11391, will be addressed in v2.4.0 once #12020 is ready to merge.
